packages/web-app-files/src/mixins/spaces/actions/editDescription.js

```diff
@@ -11,10 +11,10 @@ export default {
           name: 'editDescription',
           icon: 'pencil',
           label: () => {
-            return this.$gettext('Change description')
+            return this.$gettext('Change subtitle')
           },
           handler: this.$_editDescription_trigger,
-          isEnabled: () => false, // @TODO enable as soon as backend supports this
+          isEnabled: ({ resources }) => resources.length === 1,
```

Comment:
only enabled it there is exactly 1 resource?

Reply:
correct, also see the other actions (mixins), we will not support any batch actions on edit readme content